tests/test_raft_paper.rs

```diff
@@ -815,7 +815,7 @@ fn test_vote_request() {
         r.step(m).expect("");
         r.read_messages();
 
-        for _ in 0..r.get_election_timeout() * 2 {
+        for _ in 1..r.get_election_timeout() * 2 {
```

Comment:
Looks like we use the random number generator seeded by system, but etcd seeds it with raft id, so etcd's tests always pass.